js/utils.js

```diff
@@ -952,3 +952,39 @@ function make_SCC_graph(root_node, nodes) {
     }
     return sccs;
 }
+
+
+// Toggles display of a certain element, given the ID.
+function toggle_tab(tab) {
+    if (document.querySelector("#"+tab).style.display == "none") {
+        document.querySelector("#"+tab).style.display = "";
+    } else {
+        document.querySelector("#"+tab).style.display = "none";
```

Comment:
Performance: grab the thing by queryselector once and then use it

Also is this equivalent to `document.getElementById`? better to be explicit

Reply:
fixed